frontend/public/components/dashboards-page/overview-dashboard/top-consumers-card.tsx

```diff
@@ -114,23 +163,45 @@ const TopConsumersCard_: React.FC<DashboardItemProps> = ({
         </ConsumersFilter>
         <ConsumersBody>
           <BarChart
-            data={data}
-            query={currentQuery}
+            data={top5Data}
             title={`${type} by ${metricTypeSort.description}`}
-            loading={!topConsumersResult}
+            loading={!consumersLoadError && !(topConsumersResult && consumersLoaded)}
+            LabelComponent={({ title, metric }) => (
+              <BarLabelComponent
+                title={`${title}`}
```

Comment:
Why not simply `title={title}` ?

Reply:
`title` is `string | number | Date` so Im converting it to `string`

Reply:
OK, I see.